index.html

```diff
@@ -37,10 +38,18 @@
           </div>
         </div>
       </nav>
+      <nav class="desktop-version">
+        <ul class="nav-list">
+            <li><a class="active" href="#">Hello</a></li>
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
```

Comment:
- [ ] You may consider linking your anchor tag to their specific section on the page, so that when the user clicks a link they may be redirected to that specific section on the page.